gm4_mountaineering/data/gm4_mountaineering/functions/climbing_effects/damage_shulkers.mcfunction

```diff
@@ -0,0 +1,8 @@
+# increase the score of each climbing shulker, and kill after reaching 6
+# @s = shulker with tag gm4_mountaineering_climb_assist
+# at @s
+# run from player_main
```

Comment:
```suggestion
# run from main
```
This will remove shulkers the player is standing on, not sure if that's intended

Reply:
That behaviour is intended, since I don't think it should be possible to hang on to a wall forever; eventually, you get tired (or I do). But if it is preferred by more reviewers that a player can stand on a wall forever, that can be changed.